options.go

```diff
@@ -105,3 +109,54 @@ func WithRootCertificateFile(cert string) ClientOption {
 		o.cert = cert
 	}
 }
+
+/////////////////////////////////////////
+// Task Pool Options
+/////////////////////////////////////////
+
+type TaskPoolOptions struct {
+	tQLen      int // task queue length
+	tQNumber   int // task queue number
+	tQPoolSize int // task pool size
+}
+
+func (o *TaskPoolOptions) validate() {
+	if o.tQPoolSize == 0 {
```

Comment:
changed to `o.tQPoolSize <=0` would be better

Reply:
done. changed to `o.tQPoolSize < 1`.